### ISSUE TITLE
AA AI can't lock on all C-130 variants

### ISSUE THREAD

**Issue by rlex** [Fullerpj/United-States-Air-Force-Mod#28]
...making it completely OP in coop sessions, breaking whole game (bots simply doesn't stand a chance even if they got 100 AA units)
I really hope this can make into hotfix.
Tested with other USAF planes, seems to be fine with fighters and bombers, but all C-130 variations is affected.
Tested and affected: 
C-130 Super Hercules
AC-130, both spooky and spectre.
MC-130J Commando

**Comment by Fullerpj** [comment]
will test myself

**Comment by Fullerpj** [comment]
wont make it into hotfix sorry

**Comment by rlex** [comment]
Digged around that issue. Titan AA can lock onto AC-130 and other "problematic" planes, so it's not locking config problem. Also AA vehicles with AI inside fire on AC-130 just fine. Looks more like AI issue. What i can say about AI:

1) They are fully aware about presence of AC-130
2) They are tracking AC-130 with their weapons (even with "Stand" version of titan AA - they track plane, but do not shoot)
3) They are trying to shoot it down with small arms (lol), anti-air cannons, and even IFV cannons/machineguns.

After some testing i can say that this is also applies to C-5 Galaxy and B-1. All other seems to be fine.

Now for some weird stuff. I place F-22 which have no problems with AI on ground. Engines powered off, pilot in plane. I place CSAT AT Rifleman (with RPG-32) near that plane. As soon as he spots static plane with player inside, he uses RPG to destroy it. Tested like 10 times, always that way. Also tried C-17 which also have no AI issues, with same results. Good - this is expected outcome.

Now i place AC-130 on exact same location and with same AT unit. He will not try to use RPG - he will use his rifle instead. I tried to move plane and enemy unit around (maybe something with engagement range, i dunno), but he will never use his RPG to destroy AC-130 (or C-5). All AI will avoid using rocket launchers on that planes no matter what.

Not sure if this will help, but that's my results.
